meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+* MDJ: Question about linked issue - it sounds like a better idea to just use what is there rather than taking to office hours or take to working group
+* RG: A lot is already implemented e.g. in CSS, and there is already usage. But in implementation people found underspecified cases and need clearer requirements based on those things. We need to take another specification step to iron out those details.
+* eP: We have created a crdt4rdf CG, no activity yet. Next week planning to send out emails and start organising. Please join if you're interested. https://www.w3.org/community/crdt4rdf/
+* RG: I agree mapping should be 1 step removed. In notifs we want to make sure that we have proper security, such that if access controls are changed to a parent resource, that is properly handled when we are dealing with the person recieving the notifications (you should be able to stop those notifs from coming in). For the table, we have been iterating it, and there are a bunch of question marks there. When we are implementing this, some case comes up that we are unsure how to deal with in that moment. More about exchanging notes and making sure we are using a common format for the messages. I also agree that we should be ???
+* eP: Taking some actions - fits under LWS. Is there interest from the WG to at least see if it is fitting under the work, and making sure we are coordinating that in some way.
+* MDJ: Think we should go with the table that you created, if we need clarifications for some cases - then we should clarify. If there is a contradiction then we should discuss until we agree in the community group. Lets keep working with running code and when we run into a question like that we shouldn't wait on the WG to give us an answer as it will take 2 years. Should keep working on Solid as we have it, and make things interoperable on top of it. Will always be non-interopable things in an incubator and community, we should just document when that is the case and see if it converges. We cannot even really force anybody to converge - so should start with what people build, describe that, and then the WG can look, learn from that, and come up with a spec that hopefully aligns.
```

Comment:
```suggestion
* eP: Taking some actions — fits under LWS. Is there interest from the WG to at least see if it is fitting under the work, and making sure we are coordinating that in some way?
* MDJ: I think we should go with the table that you created. If we need clarifications for some cases — then we should clarify. If there is a contradiction, then we should discuss until we agree in the CG. Let's keep working with running code, and when we run into a question like that, we shouldn't wait on the WG to give us an answer, as it will take 2 years. Should keep working on Solid as we have it, and make things interoperable on top of it. There will always be non-interoperable things in an incubator and community; we should just document when that is the case and see if it converges. We cannot even really force anybody to converge — so should start with what people build, and describe that; then the WG can look, learn from that, and come up with a spec that hopefully aligns.
```